### ISSUE TITLE
Partie finance dans la documentation à masquer

### ISSUE THREAD

**Issue by lbranaa** [datagir/nosgestesclimat#1726]
Dans la documentation, il y a une partie sur la finance : https://nosgestesclimat.fr/documentation/finance

Est-ce qu'il ne faut pas supprimer cette partie ? 
Est-ce qu'il y a d'autres parties générées automatiquement qu'on veut masquer ?

**Comment by Clemog** [comment]
Oui, à mon avis c'est à masquer ? Ou ajouter un avertissement dans le modèle finance ?

**Comment by Clemog** [comment]
Temporairement, j'ai archivé le modèle (https://github.com/datagir/nosgestesclimat/commit/379e324e11036da6fa0d602af160560afbbd0e30). Je laisse l'issue ouverte car c'est quelque chose qu'il faudrait mieux gérer (la possibilité de publier des modèles en cours de réflexion, pas encore implémenté avec un warning -> raison de plus pour gérer la doc react de nôtre côté)

**Comment by laem** [comment]
Oui, je pense qu'il faut garder ces pages en ligne pour répondre aux utilisateurs qui n'auraient pas vu la FAQ / pour se placer en SEO sur ce mot clef. 

C'est assez aisé d'ajouter un bandeau orange : "attention, ce calcul est considéré comme de mauvaise qualité". 

![image](https://user-images.githubusercontent.com/1177762/220309620-4ab80397-237e-4f35-9d2c-9eaff1c84fb5.png)

**Comment by Clemog** [comment]
Assez aisé ? 
En détectant en dehors du composant documentation de Publicodes si l'url contient une règle non implémentée mais een ligner quand même et en affichant un message ?
Ce serait quand même pas très propre ..

**Comment by Clemog** [comment]
Un petit modèle "Livres" sera également archivé à la suite de l'ameublement

**Comment by laem** [comment]
Il faudrait que je regarde plus attentivement si on considère que c'est à prioriser, mais oui ce sera forcément un peu de hack :)

**Comment by Clemog** [comment]
De mon côté, je ne pense pas pour le moment, je notais surtout pour pas oublier
